text_mining.py

```diff
@@ -1,64 +1,81 @@
 # Markov Text Synthesis
 # David Papp
-"""
-Changes:
-- Paramaterized URL to import from. 
-- Choose whether to read text in from URL or pickle file
-- Cleaned up the dictionary creation by using the .get and .split function
-- Increased the efficiency of stripping text by using .translate
-- Removed all functions that generated Markov text by degree.
-- Added comments and improved docstrings.
-"""
```

Comment:
Not much new functionality, but good job on the fixes you made. Why did you remove Markov by degree?